source/globus/python_sdk.rst

```diff
@@ -0,0 +1,234 @@
+However, this consent was general.  
+
+To transfer data between managed collections (like those of the VSC), we have to request the user's consent for both collections specifically.
+This is done by creating a scope object to be used by the authentication process. 
+
+.. Note::
+      You only have to create scope objects for collections that require extra authentication (which is most managed collections).  
+      There are some collections to which every user has access by default:
+      
+      - Globus tutorial endpoint 1
+      - Globus tutorial endpoint 2
+      - Their own local endpoints
+      - ...
+
+      If during this section, you get an error that looks like this::
+
+            client_id=<your_client_id> requested unknown scopes: ['https://auth.globus.org/scopes/ddb59aef-6d04-11e5-ba46-22000b92c6ec/data_access']
+      
+      it is likely that you have requested a scope for a collection you didn't need it for. 
+
+
+In the section 'Authentication', we started the authentication without explictly providing a scope::
+      
+      auth_client.oauth2_start_flow()
+
+When we want to transfer from e.g. 'VSC VUB Tier2' to 'VSC KU Leuven tier2 scratch', we need to create a scope which includes these two collections.
+
+We start from a TransferScopes object and make it mutable. Then, we add consent for data access to the two collections::
+
+      from globus_sdk.scopes import GCSCollectionScopeBuilder, TransferScopes
+
+      transfer_scope = TransferScopes.make_mutable("all")
+
+      # we need the ID's of both endpoints
+      KULeuven_tier2_scratch = "872a58ab-02b9-4233-a3e0-f017ed8ab090"
+      VUB_tier2 = "2d1d4873-a849-4b9c-bd34-2034a2163003"
+
+      # adding the endpoints to the scope
+      transfer_scope.add_dependency(
+            GCSCollectionScopeBuilder(KULeuven_tier2_scratch).data_access, optional=True
+      )
+      transfer_scope.add_dependency(
+            GCSCollectionScopeBuilder(VUB_tier2).data_access, optional=True
+      )
+
+Now, we can use this scope to initiate our session::
+
+      auth_client.oauth2_start_flow(requested_scopes=[transfer_scope])
+
+Apart from this, the authentication process remains the same as seen earlier.  
+
+
+Transferring data
+----------------
+
+To transfer data from one collection to another, we first need to authenticate and require consent for the source and destination collection, as show in the sections 'Authentication' and 'Adding Scopes'.   
+ 
+Next, we instantiate an authorizer and transfer client::
+
+      authorizer = globus_sdk.AccessTokenAuthorizer(TRANSFER_TOKEN)
+      transfer_client = globus_sdk.TransferClient(authorizer=authorizer)
+
+After that, we create a transfer task::
+
+      task_data = globus_sdk.TransferData(
+            # we specify the transfer client, source collection and destination collecion
+            transfer_client, VUB_tier2, KULeuven_tier2_scratch
+      )
+
+Then, we add data to the transfer::
```

Comment:
This code will work if you are transferring 1 single file. For transferring a directory between 2 endpoints you will need to add in the add_item the parameter recursive=True or the transfer will fail with and error. For more information see: 

https://globus-sdk-python.readthedocs.io/en/stable/services/transfer.html#globus_sdk.TransferData.add_item

I think this should be explain here

Reply:
Done in my last commit